examples/vision/learnable_resizer.py

```diff
@@ -263,11 +263,13 @@ def get_model():
 combination of natural and stylized images. It might be interesting to investigate if
 this learnable resizing module could achieve something similar as the outputs seem to
 discard the texture information. 
+* The resizer module can handle arbitrary resolutions and aspect ratios which is very
+important for tasks like object detection and segmentation. 
 * Through a set of experiments, the authors also verify if the performance improvement is
 ***not*** solely due to the increase in the number of model parameters. You are
 encouraged to check those out in the
 [original paper](https://arxiv.org/abs/2103.09950v1). 
```

Comment:
I highly recommend spelling out the name of the paper here. There are at least 2 papers in this example. Therefore, "original paper" is kind of confusing and the user would have to click on it to find out what the reference is about.

Reply:
The name of the paper is too long for that purpose IMO. Hence, I think mentioning it one/two times is fine.

Reply:
Yup. IMO, you should still make a distinction w.r.t what the "original paper" means for the readers.

You have:
- [ImageNet-trained CNNs are biased towards texture; increasing shape bias improves accuracy and robustness](https://arxiv.org/abs/1811.12231)
- [Learning to Resize Images for Computer Vision Tasks](https://arxiv.org/abs/2103.09950v1)

Using their long names is not an issue, in my view, as long as you help the users.

Reply:
Since at the beginning the paper's name is spelled out, it's okay to refer to it as "original paper" IMO.

Reply:
Gotcha 👍  (the Notes section (and the majority of the example) is/are based on the idea of learned resizers from _Learning to Resize Images for Computer Vision Tasks_, which is the "original paper".)